src/VirtoCommerce.CatalogPersonalizationModule.Data/Common/ModuleConfigurator.cs

```diff
@@ -0,0 +1,96 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Hangfire;
+using VirtoCommerce.CatalogPersonalizationModule.Data.BackgroundJobs;
+using VirtoCommerce.CatalogPersonalizationModule.Data.Search.Indexing;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Settings;
+using VirtoCommerce.SearchModule.Core.Model;
+using static VirtoCommerce.CatalogPersonalizationModule.Core.ModuleConstants;
+
+namespace VirtoCommerce.CatalogPersonalizationModule.Data.Common
```

Comment:
Regarding these changes, we have some vision of how to index configuration should look like, we need to discuss before you can start the concrete implementation of this improvement.

Reply:
Separate task created https://virtocommerce.atlassian.net/browse/VP-2386